docs/INSTALLATION.MD

```diff
@@ -0,0 +1,16 @@
+# Installation steps
+Follow the steps below to set up the repo for development.
+1. Clone the repository and change the working directory
+    ```sh
+    git clone https://github.com/avantifellows/auth-layer.git
+    cd plio-frontend
+    ```
+    
+2. Set up your `.env` file by copying `.env.example` file
```

Comment:
Just also mention here the part of setting up .env files for staging and prod
`.env.staging` and `.env.production`